diff --git a/node_modules/llama.rn/llama-rn.podspec b/node_modules/llama.rn/llama-rn.podspec
index bb20b79..6a88dec 100644
--- a/node_modules/llama.rn/llama-rn.podspec
+++ b/node_modules/llama.rn/llama-rn.podspec
@@ -38,7 +38,7 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = {
     "OTHER_LDFLAGS" => base_ld_flags,
     "OTHER_CFLAGS" => base_optimizer_flags,
-    "OTHER_CPLUSPLUSFLAGS" => base_optimizer_flags + " -std=c++17"
+    "OTHER_CPLUSPLUSFLAGS" => base_optimizer_flags + " -std=c++20"
   }
 
   # Don't install the dependencies when we run `pod install` in the old architecture.
